radix-engine/tests/common_transactions.rs

```diff
@@ -0,0 +1,338 @@
+use radix_engine::ledger::TypedInMemorySubstateStore;
+use radix_engine::types::{
+    hash, Bech32Encoder, Blob, ComponentAddress, Decimal, FromPublicKey, NonFungibleAddress,
+    NonFungibleId, ResourceAddress, ACCOUNT_PACKAGE, FAUCET_COMPONENT, RADIX_TOKEN,
+};
+use radix_engine_interface::core::NetworkDefinition;
+use scrypto_unit::TestRunner;
+use transaction::manifest::compile;
+use transaction::signing::EcdsaSecp256k1PrivateKey;
+
+/// An example manifest for freeing some funds from the faucet
+#[test]
+fn free_funds_from_faucet_succeeds() {
+    test_manifest(|account_component_address, bech32_encoder| {
+        let manifest = format!(
+            include_str!("../../transaction/examples/faucet/free_funds.rtm"),
+            faucet_component_address =
```

Comment:
If you import the `ContextualDisplay` trait, you should be able to do `FAUCET_COMPONENT.to_string(bech32_encoder)` :)

Reply:
Agreed. I have just made the change to use `ContextualDisplay::display` for all addresses.